client/product-details/index.js

```diff
@@ -1,6 +1,42 @@
+/* global jQuery */
+// global wcpayStripeSiteMessaging
+
 /**
  * Internal dependencies
  */
-import bnplSiteMessaging from './bnpl-site-messaging';
+import { initializeBnplSiteMessaging } from './bnpl-site-messaging';
+
+const bnplPaymentMessageElement = initializeBnplSiteMessaging();
+
+const { productPrices } = window.wcpayStripeSiteMessaging;
+let { productId } = window.wcpayStripeSiteMessaging;
```

Comment:
Can those vars be moved inside the function?

Reply:
So those are not polluting global scope.

Reply:
I have moved it within the function.